examples/node.yaml

```diff
@@ -49,13 +49,15 @@ spec:
   - 'secret'
 ---
 kind: Deployment
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 metadata:
   name: kube-stresscheck
-  namespace: kube-system
```

Comment:
Why do you remove the namespace? we need to define it to make the RBAC work properly

Reply:
I thought it wasn't conditionned by the namespace, as executing it without namespace in Deployment definition didn't raise an error. This way I was able to put the Deployment in another namespace.
I can revert it back if you prefer.

Reply:
yeah the problem is the pod security policy binding is tied to the `kube-system` namespace
https://github.com/giantswarm/kube-stresscheck/blob/master/examples/node.yaml#L22
:)

Reply:
Fixed !